public/practice2/practice2.css

```
@@ -0,0 +1,37 @@
+.container{
```

ここはスペース欲しいです！
他にも見受けられるので確認お願いします！
```suggestion
.container {
```